node/src/components/consensus/era_supervisor.rs

```diff
@@ -370,6 +373,22 @@ where
     pub(crate) fn stop_for_upgrade(&self) -> bool {
         self.stop_for_upgrade
     }
+
+    fn executed_block(&mut self, block_header: &BlockHeader) {
```

Comment:
Can you add a comment? I got a bit lost between `next_executed_block`, `next_executed_height` :dizzy_face:

Reply:
Done.